src/test/java/com/example/demo/model/BookTest.java

```diff
@@ -0,0 +1,42 @@
+package com.example.demo.model;
+
+import com.example.demo.model.util.Language;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.Assert.assertTrue;
+
+public class BookTest {
+
+    private Book book;
+    private String bookInfo;
+
+    @Before
+    public void setUp() {
+        book = new Book("Harry Potter and the Philosopher's Stone", "J.K. Rowling",
```

Comment:
To chcialbym, zeby przetestowac getery i setery, wystarczy jeden wybrany dla praktyki. W Book.class zrob jeszcze konstruktor bez argumentów i w kazdym tescie tworzymy nowe obiekty i testujemy czy sie poprawni zachowuja.

Warto tez podzielic test na trzy fazy komentarzami //given //when //then. Latwiej sie potem to analizuje